cloud/jenkins/pgo-v2-operator-aws-openshift-latest.yml

```diff
@@ -0,0 +1,46 @@
+- job:
+    name: pgo-v2-operator-aws-openshift-latest
```

Comment:
All other v2 jobs are called without this addition, so let's keep it consistent.